src/Controller/AppController.php

```diff
@@ -32,6 +32,13 @@
  */
 class AppController extends Controller
 {
+    /**
+     * Object type currently used
+     *
+     * @var string
+     */
+    protected $objectType = null;
```

Comment:
we could probably add `$objectType` to `CategoriesController` like we did in `BulkController` and not define in `AppController` as it will be used by just 3 controllers.